doc/media/openage/sprite_format_spec.md

```diff
@@ -0,0 +1,315 @@
+
+All attributes start with a defined keyword followed by parameter values. Some
+parameters have default values and are optional. The preferred file extension is
+`.sprite`.
+
+
+## Quick Reference
+
+```
+# This is a sprite configuration file
+# comments start with # and are ignored
+
+# file version
+version <version_no>
+
+# image file reference, relative to this file's location
+imagefile <image_id> <filename>
+
+# the zoom level at which the animation is shown in full detail
+# e.g. scalefactor 2 -> full detail at 200% zoom
+scalefactor 1
+
+# layer definitions
+# all layers will be drawn
+layer <id> mode=off  position=<int>
+layer <id> mode=once position=<int> time_per_frame=<float>
+layer <id> mode=loop position=<int> time_per_frame=<float> replay_delay=<float>
+
+# define an angle where frames can be assigned to or mirror from an existing angle
+angle <degree> mirror-from=<existing_angle>
+
+# assign frames to their layers and angles.
+# angle is the direction in degrees, etc.
+# *pos, *size and *hotspot is within the source image.
+# all the hotspots of the frames will be drawn at the same pixel (requested by renderer)
+# so that the alignment/movement of the frames is done solely by hotspots.
+frame <id> <angle> <layer_id> <image_id> <xpos> <ypos> <xsize> <ysize> <xhotspot> <yhotspot>
+```
+
+
+## Data Type Formatting
+
+Type     | Example | Description
+---------|---------|---------
+int      | `5`     | Signed Integer
+float    | `1.2`   | Float
+string   | `"bla"` | String of characters enclosed by `"`
+token    | `off`   | Alphanumeric predefined keyword
+
+
+## Attributes
+
+### `version`
+
+Version of the sprite format. Increments every time the syntax
+or keywords of the format change.
+
+Parameter  | Type   | Optional | Default value
+-----------|--------|----------|--------------
+version_no | int    | No       | -
+
+**version_no**</br>
+Version number of the format.
+
+#### Example
+
+```
+version 1
+```
+
+
+### `imagefile`
+
+Tells the renderer which image resources it has to load as spritesheets.
+There has to be at least one `imagefile` defined.
+
+Parameter | Type   | Optional | Default value
+----------|--------|----------|--------------
+image_id  | int    | No       | -
+filename  | string | No       | -
+
+**image_id**</br>
+Reference ID for the resource used in this file. IDs should start at `0`.
+
+**filename**</br>
+Path to the image resource on the filesystem.
+
+There are two ways to specify a path: relative and absolute. Relative
+paths are relative to the location of the sprite definition file. They
+can only refer to image resources that are in the same modpack.
+
+```
+imagefile 0 "idle.png"         # idle.png is in the same folder as the sprite file
+imagefile 1 "./idle.png"       # same as above, but more explicit
+imagefile 2 "media/attack.png" # attack.png is in the subfolder media/
+```
+
+Absolute paths start from the modpack root folder (the path where all
```

Comment:
Renamed it to "(virtual) modpack root". I would not use "openage" here because the openage folders could be at different locations, e.g. binaries (probably `/usr/share`) and user data (probably somewhere in `~`)